src/OpenSage.Game/Logic/Object/Draw/W3dHordeModelDraw.cs

```diff
@@ -9,6 +10,39 @@ public class W3dHordeModelDrawModuleData : W3dModelDrawModuleData
         internal static W3dHordeModelDrawModuleData Parse(IniParser parser) => parser.ParseBlock(FieldParseTable);
 
         internal static new readonly IniParseTable<W3dHordeModelDrawModuleData> FieldParseTable = W3dModelDrawModuleData.FieldParseTable
-            .Concat(new IniParseTable<W3dHordeModelDrawModuleData>());
+            .Concat(new IniParseTable<W3dHordeModelDrawModuleData>
+            {
+                { "StaticModelLODMode", (parser, x) => x.StaticModelLODMode = parser.ParseBoolean() },
+                { "LodOptions", (parser, x) => x.LODOptions.Add(LODOption.Parse(parser)) }
+            });
+
+        public bool StaticModelLODMode { get; internal set; }
+        public List<LODOption> LODOptions { get; internal set; } = new List<LODOption>();
+    }
+
+    public sealed class LODOption
+    {
+        internal static LODOption Parse(IniParser parser)
+        {
+            var lod = parser.ParseEnum<ModelLevelOfDetail>();
+            var result = parser.ParseBlock(FieldParseTable);
+            result.LOD = lod;
+            return result;
+        }
+
+        internal static readonly IniParseTable<LODOption> FieldParseTable = new IniParseTable<LODOption>
+            {
+                { "AllowMultipleModels", (parser, x) => x.AllowMultipleModels = parser.ParseBoolean() },
+                { "MaxRandomTextures", (parser, x) => x.MaxRandomTextures = parser.ParseInteger() },
+                { "MaxRandomAnimations", (parser, x) => x.MaxRandomAnimations = parser.ParseInteger() },
+                { "MaxAnimFrameDelta", (parser, x) => x.MaxAnimFrameDelta = parser.ParseInteger() }
+            };
+
+        public ModelLevelOfDetail LOD { get; internal set; }
+
+        public bool AllowMultipleModels { get; internal set; }
```

Comment:
Do these need to be `internal set`? Could they be `private set`? I've noticed that in a few other places too.

Reply:
Saw that in a few places, and used internal by default for new objects. Should I use private instead?

Reply:
I thought I only used `internal` when it actually needed to be accessed by something else, like a subclass or another class. Yes, `private` is better if it doesn't need to be accessed by anything else.

Reply:
ok, so should i change that for all the objects, also for the ones from the old PR?

Reply:
If you have the energy to do so, yes please.